pkg/promtail/scrapeconfig/scrapeconfig_test.go

```diff
@@ -97,13 +99,18 @@ func TestLoadSmallConfig(t *testing.T) {
 	err := yaml.Unmarshal([]byte(smallYaml), &config)
 	require.Nil(t, err)
 
+	fmt.Printf("%#v\n", config.ServiceDiscoveryConfig.KubernetesSDConfigs)
```

Comment:
```suggestion
```

Reply:
Sorry, I was trying to find out where the FollowRedirects field was set and did some printf debugging.